src/validator.ts

```diff
@@ -1,23 +1,53 @@
 import { FieldValidationFunctionSync } from '@lemoncode/fonk';
 
-// TODO: Add validator type
-const VALIDATOR_TYPE = '';
+const VALIDATOR_TYPE = 'MIN_NUMBER';
 
-// TODO: Add default message
 let defaultMessage = '';
-export const setErrorMessage = message => (defaultMessage = message);
+export const setErrorMessage = (message: string) => (defaultMessage = message);
+
+const createCustomMessage = (customArgs: CustomValidatorArgs) =>
```

Comment:
Mmm.. how can we solve this if we override the error message?

Reply:
I think more sooner than later we will have to define our own interpolation stuff like i18next does, what do you think?